change/office-ui-fabric-react-2019-08-06-13-55-21-lorejoh12-fixShimmer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "moving the async clearTimeout call inside the if check to prevent accidentally clearing the timeout unless we're going to be making a new one",
```

Comment:
```suggestion
  "comment": "Shimmer: moving the async clearTimeout call inside the if check to prevent accidentally clearing the timeout unless we're going to be making a new one.”,
```